src/rete/controls/LabeledInputControl.tsx

```diff
@@ -1,5 +1,63 @@
 import { ClassicPreset as Classic } from 'rete'
```

Comment:
Getting a few warnings about the imports (and also line 32) that we can resolve easily
![image](https://github.com/WebAudio-Node-Editor/webaudio-node-editor/assets/55116848/a2f7e2cb-0c82-40ee-b2fe-cd3ff00775c7)